### ISSUE TITLE
RFE: allow a plugin to render a link to another tool from the node search result

### ISSUE THREAD

**Issue by fenner** [netdisco/netdisco#1118]
At the IETF we have a new tool that can deliver information about a wireless client based on MAC address.  Netdisco is really great at finding clients by IP and delivering the MAC address.  I'd love a way to provide a plugin that augments to the node search results page with a link directly to the tool that provides the new information.

(Looking at `views/ajax/search/node_by_mac.tt` I guess for flexibility I'd like to be able to inject for MAC or IP or IP6, just thinking about future possibilities)

**Comment by ollyg** [comment]
Hi @fenner would it need to be a plugin (with some code running to generate the link) or sufficient to do via config with a template string, for example:

```
hyperlinks:
  node:
    - link: https://my.wifi.manager.example.com/search?node=[% mac %]
      displayname: 'WiFi Manager'
      for_mac: true
```

**Comment by fenner** [comment]
> would it need to be a plugin

I just meant plugin metaphorically - configuration is fine.

**Comment by ollyg** [comment]
implemented in ac54635
